src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -103,6 +103,11 @@ private set
     /// </summary>
     private JoinableTask<IInlineRenameLocationSet> _allRenameLocationsTask;
 
+    /// <summary>
+    /// Expose <see cref="_allRenameLocationsTask"/>
+    /// </summary>
+    internal JoinableTask<IInlineRenameLocationSet> AllRenameLocationsTask => _allRenameLocationsTask;
```

Comment:
why?

Reply:
previous investigation. I've removed this.